Fix missing map in replay mode (#1320)
hpi-sam/digital-fuesim-manv: CHANGELOG.md

### Fixed

- Viewports are now consistently called "Ansicht" in the frontend and some other incorrect usages of "Einsatzabschnitt" have been corrected.

> - In the replay mode, the map was missing.

Reviewer: ```suggestion
- The map is now visible again in replay mode.
```
